test/backend/test_node.py

```diff
@@ -121,7 +121,7 @@ def _batch_normalization(self, x, mean, variance, bias, scale,
     return x * inv + (bias - mean * inv if bias is not None else -mean * inv)
 
   def test_batch_normalization(self):
-    if defs.onnx_opset_version() < 6:
+    if legacy_opset_pre_6:
```

Comment:
.. lack of parentheses

Reply:
I think I fixed that immediately in the commit named "typo".